torch/nn/qat/modules/linear.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from ...modules.linear import Linear as NNLinear
+from torch.quantization.QConfig import default_qat_qconfig
+import torch.nn.functional as F
+
+class Linear(NNLinear):
+    r"""
+    A linear module attached with FakeQuantize modules for both output
+    activation and weight, used for quantization aware training.
+
+    We adopt the same interface as `torch.nn.Linear`, please see
+    https://pytorch.org/docs/stable/nn.html#torch.nn.Linear
+    for documentation.
+
+    Similar to `torch.nn.Linear`, with FakeQuantize modules initialized to
+    default.
+
+    Attributes:
+        observer: fake quant module for output activation, it's called observer
+            to align with post training flow
+        weight: fake quant module for weight
+
+    Examples::
+
+        >>> m = nn.qat.Linear(20, 30)
+        >>> input = torch.randn(128, 20)
+        >>> output = m(input)
+        >>> print(output.size())
+        torch.Size([128, 30])
+    """
+    __constants__ = ['bias', 'in_features', 'out_features']
+
+    def __init__(self, in_features, out_features, bias=True,
+                 activation_fake_quant=default_qat_qconfig.activation(),
```

Comment:
is there any value in keeping the signature the same as `nn.Linear`? this module is probably used only temporarily when we do qat and I think we don't really expect user to interact with it.